src/GlobalHeader/index.js

```diff
@@ -134,280 +142,267 @@ const GlobalHeader = ({ globalNav, pages, docs, location, ...props }) => {
+                tabs.push(ref)
 
-                  const isActive = ({ isPartiallyCurrent }) =>
-                    isPartiallyCurrent
-                      ? {
-                          'aria-selected': 'true',
-                          className: 'is-selected spectrum-Tabs-item'
-                        }
-                      : { className: 'spectrum-Tabs-item' }
+                const isActive = ({ isPartiallyCurrent }) =>
+                  isPartiallyCurrent
+                    ? {
+                        'aria-selected': 'true',
+                        className: 'is-selected spectrum-Tabs-item'
+                      }
+                    : { className: 'spectrum-Tabs-item' }
 
-                  return (
+                return (
+                  <React.Fragment key={i}>
                     <GatsbyLink
                       role='tab'
                       getProps={isActive}
-                      key={index}
                       ref={ref}
                       to={path}
                       partiallyActive
                     >
                       <span className='spectrum-Tabs-itemLabel'>{title}</span>
                     </GatsbyLink>
-                  )
-                })}
-              <div
+                    {i === 0 && versions[0]?.title && (
+                      <div
+                        css={css`
+                          margin-left: var(
+                            --spectrum-global-dimension-static-size-100
+                          ) !important;
+                          margin-right: var(
+                            --spectrum-global-dimension-static-size-300
+                          );
+                        `}
+                      >
+                        <Picker
+                          isQuiet
+                          items={versions.map((version, k) => ({
+                            title: version.title,
+                            path: version.path,
+                            selected: k === 0
+                          }))}
+                        />
+                      </div>
+                    )}
+                  </React.Fragment>
+                )
+              })}
+              <TabsIndicator
                 ref={selectedTabIndicator}
-                className='spectrum-Tabs-selectionIndicator'
                 css={css`
                   bottom: -10px !important;
-                  transition-property: transform, width;
                 `}
               />
               <View marginStart='size-400'>
-                {docs?.path && (
+                {docs.path && (
                   <Button variant='primary' elementType='a' href={docs.path}>
                     View Docs
                   </Button>
                 )}
               </View>
-            </div>
+            </Tabs>
           </View>
           <View gridArea='console' justifySelf='center'>
-            {globalNav?.console && (
+            {globalNav.console && (
```

Comment:
Can this be a prop instead of a part of the `globalNav` input?

Reply:
yes let me know what works best for you